### PR TITLE
[Merged by Bors] - doc(Data/Multiset/Basic): write a proper module doc

### PR DESCRIPTION
**Pull request by YaelDillies**
---

[Open in Gitpod](https://gitpod.io/from-referrer/)

### CODE REVIEW

File: Mathlib/Data/Multiset/Basic.lean [hunk 1]
```diff
@@ -14,9 +14,20 @@ import Mathlib.Order.Hom.Basic
...
+Multisets are finite sets with duplicates allowed. They are implemented here as the quotient of
+lists by permutation. This gives them computational content.
+
 ## Notation
-We define the global infix notation `::ₘ` for `Multiset.cons`.
+
+* `0`: The empty multiset.
+* `{a}`: The multiset containing a single occurrence of `a`.
+* `a ::ₘ s`: The multiset containing one more occurrence of `a` than `s`.
+* `s + t`: The multiset containing for each `a` the sum of the occurrences of `a` in `s` and `t`.
```
Comment (b-mehta) [inline Mathlib/Data/Multiset/Basic.lean:26]: This phrasing feels very unnatural to me... 
```suggestion
* `s + t`: The multiset for which `count a (s + t) = count a s + count a t`.
```
or
```suggestion
* `s + t`: The multiset for which the number of occurrences of `a` is sum of the occurrences of `a` in `s` and `t`.
```
(and similarly below)

File: Mathlib/Data/Multiset/Basic.lean [hunk 2]
```diff
@@ -14,9 +14,20 @@ import Mathlib.Order.Hom.Basic
...
+
+Multisets are finite sets with duplicates allowed. They are implemented here as the quotient of
+lists by permutation. This gives them computational content.
+
 ## Notation
-We define the global infix notation `::ₘ` for `Multiset.cons`.
+
+* `0`: The empty multiset.
+* `{a}`: The multiset containing a single occurrence of `a`.
+* `a ::ₘ s`: The multiset containing one more occurrence of `a` than `s`.
```
Comment (b-mehta) [inline Mathlib/Data/Multiset/Basic.lean:25]:
```suggestion
* `a ::ₘ s`: The multiset containing one more occurrence of `a` than `s` has.
```
(else a valid interpretation would be that the multiplicity of `a` is one more than the multiplicity of `s`

Comment (YaelDillies) [reply Mathlib/Data/Multiset/Basic.lean]: What about now? Not sure I am speaking the english very good

**Review by b-mehta — COMMENTED** [review]
Thanks!

bors merge

**Pull request closed without merging**